hazelcast/src/main/java/com/hazelcast/map/impl/operation/GetAllOperation.java

```diff
@@ -72,8 +71,7 @@ protected void evict() {
         if (recordStore == null) {
             return;
         }
-        final long now = Clock.currentTimeMillis();
-        recordStore.evictEntries(now);
+        recordStore.evictEntries();
```

Comment:
I know this behaviour is not introduced in this PR but why we trigger eviction on get all?